demo/api/src/form-builder/entities/form-builder.entity.ts

```diff
@@ -0,0 +1,66 @@
+import { BlockDataInterface, RootBlock, RootBlockEntity } from "@comet/blocks-api";
+import { CrudGenerator, RootBlockDataScalar, RootBlockType } from "@comet/cms-api";
+import { BaseEntity, Collection, Embeddable, Embedded, Entity, OneToMany, OptionalProps, PrimaryKey, Property } from "@mikro-orm/core";
+import { Field, ID, InputType, ObjectType } from "@nestjs/graphql";
+import { IsString } from "class-validator";
+import { v4 } from "uuid";
+
+import { FormBuilderBlock } from "../blocks/form-builder.block";
+import { FormBuilderInterface } from "../types";
+import { FormRequest } from "./form-request.entity";
+
+@Embeddable()
+@ObjectType("")
```

Comment:
```suggestion
@ObjectType()
```

This shouldn't be necessary